.github/workflows/push-api-docs.yaml

```diff
@@ -0,0 +1,52 @@
+# When we push a new tag to the repository, open a Pull Request to synchronize
+# the API reference documentation with the "docs" repository.
+
+name: "Push API docs"
+
+concurrency:
+  group: "ci-${{ github.ref_name }}"
+  cancel-in-progress: true
+
+on:
+  push:
+    tags:
+      - 'v*'
+
+jobs:
+  push-api-docs:
+    runs-on: "ubuntu-latest"
+    steps:
+      - name: "Checkout 'fabric' repository"
+        uses: "actions/checkout@v4"
+        with:
+          fetch-depth: 0
+          persist-credentials: "false"
+
+      - name: "Checkout 'docs' repository"
+        uses: "actions/checkout@v4"
+        with:
+          repository: "githedgehog/docs"
+          path: "project_documentation"
+          persist-credentials: "false"
+
+      - name: "Copy API reference from 'fabric' to 'docs'"
+        run: |
+          cp docs/api.md project_documentation/docs/reference/api.md
+
+      - name: "Create Pull Request"
+        uses: "peter-evans/create-pull-request@v5"
+        with:
+          token: "${{ secrets.DOCS_API_REF_UPDATE_TOKEN }}"
```

Comment:
This is what likely requires a change, and for which I would welcome suggestions

Reply:
Yeah, I guess there is no other way than managing some token in the runner secrets with permissions to create PRs

Reply:
@Frostman So I just push a new commit, although I've not tested it. This new commit adds a step to create a GitHub App token, based on the following docs:

- https://github.com/peter-evans/create-pull-request/blob/main/docs/concepts-guidelines.md#authenticating-with-github-app-generated-tokens
- https://github.com/actions/create-github-app-token

It does not work yet, because we first need to register a new dedicated GitHub App (“Hoglet” sounds like a nice App name), and store the App ID and secret key. I don't have permissions for that, can you help me please?